cpp/include/cudf/lists/detail/dremel.hpp

```diff
@@ -58,7 +58,7 @@ struct dremel_data {
 };
 
 /**
- * @brief Get the dremel offsets and repetition and definition levels for a LIST column
+ * @brief Get the dremel offsets, and repetition and definition levels for a LIST column
```

Comment:
```suggestion
 * @brief Get the dremel offsets, repetition, and definition levels for a LIST column
```

I feel like we're Oxford comma people, right?

Reply:
Yes we are!

Reply:
Doesn't it change the meaning? AFAICT, this now needs to be "dremel offsets, repetition levels, and definition levels"

Reply:
Yes it does and that's exactly what I updated it to!